app/models/planting.rb

```diff
@@ -137,4 +124,31 @@ def percentage_grown(current_date = Date.current)
 
     percent
   end
+
+  def start_to_finish_diff
+    (finished_at - planted_at).to_i
+  end
+
+  def self.mean_days_until_maturity(plantings)
+    ## Given a set of finished plantings, calculate the average/mean time from start to finish
+    differences = plantings.collect(&:start_to_finish_diff)
+
+    return if differences.compact.empty?
+
+    differences.compact.sum / differences.compact.size unless differences.compact.empty?
```

Comment:
i've left this as average (mean).
but i think it should change to median - so one user's 1600 day old tomatoes don't runs the stats for everyone else.